firefox_ui_tests/functional/locationbar/test_escape_autocomplete.py

```diff
@@ -0,0 +1,58 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness.decorators import skip_under_xvfb
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestEscapeAutocomplete(FirefoxTestCase):
+
+    def tearDown(self):
+        autocompleteresults = self.browser.navbar.locationbar.autocomplete_results
+
+        if autocompleteresults.is_open:
+            autocompleteresults.close(force=True)
+            self.wait_for_condition(not autocompleteresults.is_open)
+
+        FirefoxTestCase.tearDown(self)
+
+    @skip_under_xvfb
+    def test_escape_autocomplete(self):
+        # Clear complete history so there's no interference from previous entries.
+        self.places.remove_all_history()
+
+        # Open some local pages to set up the test environment.
+        self.test_urls = [
+            'layout/mozilla.html',
+            'layout/mozilla_community.html',
+        ]
+        self.test_urls = [self.marionette.absolute_url(t)
+                          for t in self.test_urls]
+
+        def load_urls():
+            with self.marionette.using_context('content'):
+                for url in self.test_urls:
+                    self.marionette.navigate(url)
+        self.places.wait_for_visited(self.test_urls, load_urls)
+
+        # Set variables
```

Comment:
Lets get this comment removed. I think its clear what you do here. Comments are usually helpful to describe a block of actions or something which is not immediately understandable.